### ISSUE TITLE
env: node\r: No such file or directory (I suggest changing the sequelize-auto.js file from CRLF to LF)

### ISSUE THREAD

**Issue by kimud6003** [sequelize/sequelize-auto#475]
The sequelize-auto/bin/seuqelize-auto.js is written as CRLF 

so an error such as env: node\r: No search file or directory occurs.

I solved it by changing sequelize-atuo.js to LF file.

![스크린샷 2020-12-31 오후 10 26 50](https://user-images.githubusercontent.com/61041800/103412206-54477680-4bb7-11eb-8386-46edaae8ba74.png)

**Comment by steveschmitt** [comment]
What throws the error?  Your shell, or node itself?

**Comment by kimud6003** [comment]
the error is "env: node\r: No such file or directory env: node\r: No such file or directory"
also It occurs shebang such as #!/usr/bin/env node in the first line of a file but with CRLF line endings.
![스크린샷 2021-01-06 오후 10 22 08(2)](https://user-images.githubusercontent.com/61041800/103772935-a3851f80-506d-11eb-83ff-217e2d303085.png)
This is the screen that appeared when I typed the command to run the module.
So I changed the CRLF to LF and executed it.
